### DISCUSSION TITLE
a lot of buttons

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7695] [Q&A]
I have a lot of sprits & containers which use "interactive=true; buttonMode=true".
They are around 2000~3000 elements.
Is it ok?
Should I use another way? 
I don't see any performance problem currently, but I'm just curious.

**Comment by miltoncandelero** [comment]
do you need that many elements to be clickable?
If you do there is not much to do about it.
You could try making clusters in containers to make some sort of quadtree since the interaction manager would first check if the pointer is inside the parent of an interactive object and if it is not it will skip all its descendants. 🤔
